packages/esm-patient-chart-app/src/visit-header/visit-header.component.tsx

```diff
@@ -238,18 +240,18 @@ const VisitHeader: React.FC = () => {
 
     return null;
   }, [
-    hasActiveVisit,
-    isSideMenuExpanded,
-    onClosePatientChart,
+    showVisitHeader,
     patient,
     showHamburger,
-    showVisitHeader,
+    isSideMenuExpanded,
+    isDeceased,
     startVisitLabel,
     t,
-    toggleSideMenu,
+    currentVisit,
     endVisitLabel,
+    onClosePatientChart,
+    toggleSideMenu,
     openModal,
-    currentVisit,
```

Comment:
I'd prefer things if we didn't both re-order this and remove things from it at the same time. It's really hard to review this kind of change.